ColmanGym/ColmanGym/wwwroot/graph_data/CountMeetingbyType.csv

```diff
@@ -0,0 +1,3 @@
+Training_type,AmountOfMeetings
```

Comment:
In real project it will be fetched from db?

Reply:
Yes. When you click on "meetings" it will activate the meeting controller ctr and will inital the state of the zip file from the db.
I cleared the data of the csv files in my pr